docs/content/direct/start-from-ocm.md

```diff
@@ -0,0 +1,143 @@
+# Adding KubeStellar to OCM
+
+This page shows one concrete example of adding KubeStellar to an
+existing OCM system. In particular, a hub plus two managed clusters
+almost exactly as created by the OCM Quick Start.  In terms of the
+[full Installation and Usage outline of
+KubeStellar](user-guide-intro.md#the-full-story), the modified OCM Quick Start
+has already: established some, but not all, of the software
+prerequisites; acquired the ability to use a Kube cluster as KubeFlex
+hosting cluster; created an Inventory and Transport Space; created two
+Workload Execution Clusters (WECs) and registered them.
+
+  1. [Setup](#setup)
+    1. Install remaining software prerequisites
+    1. Cleanup from previous runs
+    1. OCM Quick Start with Ingress
+    1. Label WECs for selection by examples
+    1. Install Kubestellar core components
+  2. [Exercise KubeStellar](#exercise-kubestellar)
+  3. [Troubleshooting](#troubleshooting)
+
+## Setup
+
+Continuing with the spirit of the OCM Quick Start, this is one way to
+produce a very simple system --- suitable for study but not production
+usage. For general setup information, see [the full
+story](user-guide-intro.md#the-full-story).
+
+### Install software prerequisites
+
+The following command will check for the prerequisites that you will need for the later steps. See [the prerequisites doc](pre-reqs.md) for more details.
+
+```shell
+bash <(curl https://raw.githubusercontent.com/kubestellar/kubestellar/v{{ config.ks_latest_regular_release }}/hack/check_pre_req.sh) kflex ocm helm kubectl docker kind
+```
+
+### Cleanup from previous runs
+
+If you have run this recipe or used `kflex` (the KubeFlex command
+line) in any other capacity, it may have some troublesome hidden state
+in your kubeconfig file. The following command clears out that hidden
+state.
+
+```shell
+yq -i 'del(.preferences)' ${KUBECONFIG:-$HOME/.kube/config}
+```
+
+### Set the Version appropriately as an environment variable
+
+```shell
+export KUBESTELLAR_VERSION={{ config.ks_latest_release }}
+```
+
+### OCM Quick Start with Ingress
+
+This recipe uses a modified version of [the OCM Quick Start](https://raw.githubusercontent.com/open-cluster-management-io/OCM/main/solutions/setup-dev-environment/local-up.sh). The modification is because KubeStellar needs the hosting cluster to have an Ingress controller with SSL passthrough enabled. The modified Quick Start script has the following modifications compared to the baseline.
+
+1. The `kind` cluster created for the hub has an additional port mapping, where the Ingress controller listens.
+1. The script installs an Ingress controller (based on nginx) into the hub cluster, then patches it to enable SSL passthrough, and later waits for it to be in service.
```

Comment:
best just say "NGINX Ingress Controller" which is well known (see https://docs.nginx.com/nginx-ingress-controller/ ) rather than "Ingress controller (based on nginx) "

Reply:
Done, thanks.